lib/puppet/provider/grafana_datasource/grafana.rb

```diff
@@ -68,6 +68,7 @@ def datasources
           name: datasource['name'],
           url: datasource['url'],
           type: datasource['type'],
+          version: resource[:version],
```

Comment:
Is this correct? Shouldn't this be returning the datasource as returned by the API?

ie.
```suggestion
          version: datasource[:version],
```
??

Reply:
yes this shoud be datasource, I updated the pullrequest.

Reply:
I will also provide a test for this, feature